src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayOfRent = 40;
+  let sum = 0;
+
+  if (days >= 7) {
+    sum = dayOfRent * days - 50;
+  } else if (days >= 3) {
+    sum = dayOfRent * days - 20;
+  } else {
+    sum = dayOfRent * days;
+  }
+
+  return sum;
```

Comment:
```suggestion
  } 
  
  return sum = dayOfRent * days;
```

You don't need the last 'else' because if the previous two checks won't pass, the function will return the last case by default